pkg/arvo/app/group-hook.hoon

```diff
@@ -184,35 +157,32 @@
     ?.  =(src.bol u.ship)
       [~ this]
     :_  this(synced (~(del by synced) pax.diff))
-    :~  (group-poke pax.diff diff)
-    ==
+    [(group-poke pax.diff diff)]~
   ::
   ==
 ::
 ++  quit
   |=  wir=wire
   ^-  (quip move _this)
-  =/  wir  `(list @tas)`wir
-  =/  =ship  (slav %p &1:wir)
-  =.  wir  ?^  wir  t.wir  ~
-  =.  wir  ?^  wir  t.wir  ~
-  ?:  (~(has by synced) wir)
-    =/  group-path  [%group wir]
-    =/  group-wire  [(scot %p ship) group-path]
-    :_  (track-bone group-wire)
-    [ost.bol %peer group-wire [ship %group-hook] group-path]~
-  ::  no-op
-  [~ this]
+  =^  =ship  wir
+    ?>  ?=([* ^] wir)
+    [(slav %p i.wir) t.t.wir]
+  ?.  (~(has by synced) wir)
+    ::  no-op
```

Comment:
this comment seems unnecessary, given how many no-ops there are.

Reply:
Good point, thanks.